app/controllers/admin/principals_controller.rb

```diff
@@ -8,6 +8,19 @@ def show
     @principal = principal
   end
 
+  def destroy
```

Comment:
Hey @munckymagik, Is there sufficient authorisation check to ensure that it is only the admin who can destroy a principle please?